.github/workflows/fork-sync.yaml

```
@@ -0,0 +1,24 @@
+name: Sync Fork
+
+on:
+  schedule:
+    - cron: "0 0 * * *" # once a day
```

Let's keep the sync weekly.
```suggestion
    - cron: "0 0 * * 1" # once a week on Monday
```